modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/exportimport/data/handler/JournalFeedStagedModelDataHandler.java

```diff
@@ -306,8 +306,10 @@ protected void doImportStagedModel(
 	@Reference
 	private JournalCreationStrategy _journalCreationStrategy;
 
-	@Reference
-	private JournalFeedExportImportContentProcessor
+	@Reference(
+		target = "(component.name=com.liferay.journal.internal.exportimport.content.processor.JournalFeedExportImportContentProcessor)"
```

Comment:
@daiazius @rafaprax Please use "model.class.name=com.liferay.journal.model.JournalFeed" as filter rather than component name, we only use component name when you do not have other way to identify a component.